src/pyflex/window_selector.py

```diff
@@ -911,12 +992,36 @@ def plot(self, filename=None):
         ax.spines['right'].set_color('none')
         ax.spines['left'].set_color('none')
         ax.spines['top'].set_color('none')
+        stepsize = 200
+        start, end = ax.get_xlim()
+        ax.xaxis.set_ticks(np.arange(start, end+1, stepsize))
```

Comment:
Hi Lion,

See the picture I just made.

![aak ii 00 lhz](https://cloud.githubusercontent.com/assets/5777547/10611461/3a8e8070-771a-11e5-9c2d-dfcbcc0b11f8.png)